Update gentoo builds

mookie- opened the pull request:
These are some improvements regarding building gentoo with image-bootstrap:

* The linux kernel kvm_config changed (see: https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git/tree/kernel/configs/kvm_guest.config).

* The correct location of ACCEPT_KEYWORDS changed (see: https://wiki.gentoo.org/wiki/ACCEPT_KEYWORDS).

* The dhcpcd package got fixed after your bugreport (see: https://bugs.gentoo.org/659626).

* In my opinion 5gb is enough for a gentoo base image (the build did work fine multiple times using up <4gb).

* The location/filename of signatures changed (see: https://www.gentoo.org/news/2022/02/17/changed-signatures.html).

hartwork reviewed (changes requested):
Hi @mookie,

this is amazing work, thanks a lot for this! :+1: :+1:

One small detail seems off by accident, please see my comment below. If you could adjust that and rebase onto latest `master` (which includes Arch fixes from PR #218), this PR right here will even have fully green CI at merge time.

Thanks and best, Sebastian

mookie- commented on image_bootstrap/distros/gentoo.py:
```diff
@@ -415,14 +415,20 @@ def _configure_kernel__enable_kvm_support(self):
                 CONFIG_HYPERVISOR_GUEST=y
                 CONFIG_PARAVIRT=y
                 CONFIG_KVM_GUEST=y
+                # CONFIG_KVM_GUEST=y
```
It's done :heavy_check_mark:

mookie- commented:
Thank you very much for looking into it. I've updated the PR and rebased it against hartwork:master.
Please let me know if there is anything else to do there.

hartwork reviewed (approved):
@mookie- excellent, thank you! :+1:
